<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecogerOpciones</class>
 <widget class="QDialog" name="RecogerOpciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RecogerOpciones</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="grpInfo">
         <property name="title">
          <string>Información</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="etiquetaNombre">
              <property name="text">
               <string>Nombre :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineaEditorNombre"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="etiquetaCorreo">
              <property name="text">
               <string>E_Correo :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineaEditorCorreo"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="grpComida">
           <property name="title">
            <string>Comida</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="marcarHambu">
              <property name="text">
               <string>Hamburguesa</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="marcarFideos">
              <property name="text">
               <string>Fideos</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="marcarPan">
              <property name="text">
               <string>Pan</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="marcarHuevo">
              <property name="text">
               <string>Huevo</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="grpDeporte">
           <property name="title">
            <string>Deporte</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="marcarNadar">
              <property name="text">
               <string>Nadar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="marcarCorrer">
              <property name="text">
               <string>Correr</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="marcarBaloncesto">
              <property name="text">
               <string>Baloncesto</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="marcarBadminton">
              <property name="text">
               <string>Bádminton</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>118</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnRechazar">
           <property name="text">
            <string>Rechazar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAcceptar">
           <property name="text">
            <string>Acceptar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
